assets/report_template.Rmd

```diff
@@ -0,0 +1,1175 @@
+
+cat("\n\nTables with read count numbers and filtered abundance tables are in folder [qiime2/abundance_tables](../qiime2/abundance_tables).")
+```
+
+```{r, eval = !isFALSE(params$abundance_tables), results='asis'}
+cat("## Abundance tables\n",
+    "The abundance tables are the final data for further downstream analysis and visualisations. The tables are based on the computed ASVs and taxonomic classification, but after removal of unwanted taxa. ",
+    "Folder [qiime2/abundance_tables](../qiime2/abundance_tables) contains tap-separated files (.tsv) that can be opened by any spreadsheet software.")
+
+cat("\n\n## Relative abundance tables\n",
+    "Absolute abundance tables produced by the previous steps contain count data, but the compositional nature of 16S rRNA amplicon sequencing requires sequencing depth normalisation. This step computes relative abundance tables using TSS (Total Sum Scaling normalisation) for various taxonomic levels and detailed tables for all ASVs with taxonomic classification, sequence and relative abundance for each sample. Typically used for in depth investigation of taxa abundances. ",
+    "Folder [qiime2/rel_abundance_tables](../qiime2/rel_abundance_tables) contains tap-separated files (.tsv) that can be opened by any spreadsheet software.")
+```
+
+```{r, eval = !isFALSE(params$barplot), results='asis'}
+cat("## Barplot\n",
+    "Interactive abundance plot that aids exploratory browsing the discovered taxa and their abundance",
+    "in samples and allows sorting for associated meta data.",
+    "Folder [qiime2/barplot](../qiime2/barplot) contains barplots, click [qiime2/barplot/index.html](../qiime2/barplot/index.html) to open it in your web browser.")
+```
+
+```{r, eval = !isFALSE(params$metadata_category_barplot), results='asis'}
+cat("\n\nAdditionally, barplots with average relative abundance values were produced for",
+    params$metadata_category_barplot,"(comma separated if several) in [qiime2/barplot_average](../qiime2/barplot_average)
+    in separate folders following the scheme 'barplot_{treatment}':\n")
+metadata_category_barplot <- sort( unlist( strsplit( params$metadata_category_barplot,"," ) ) )
+for (category in metadata_category_barplot) {
+    barplot_folder_path <- paste0("qiime2/barplot_average/barplot_",category)
+    cat("\n- [",barplot_folder_path,"/index.html](../",barplot_folder_path,"/index.html)\n", sep="")
+}
+```
+
+```{r, eval = !isFALSE(params$alpha_rarefaction), results='asis'}
+cat("## Alpha diversity rarefaction curves\n",
+    "Produces rarefaction plots for several alpha diversity indices, and is primarily used to determine if the richness of the samples has been fully observed or sequenced. If the slope of the curves does not level out and the lines do not becomes horizontal, this might be because the sequencing depth was too low to observe all diversity or that sequencing error artificially increases sequence diversity and causes false discoveries. ")
```

Comment:
```suggestion
    "Produces rarefaction plots for several alpha diversity indices, and is primarily used to determine if the richness of the samples has been fully observed or sequenced. If the slope of the curves does not level out and the lines do not become horizontal, this might be because the sequencing depth was too low to observe all diversity or that sequencing error artificially increases sequence diversity and causes false discoveries. ")
```